packages/connectivity/src/scp-cf/jwt.spec.ts

```diff
@@ -23,8 +23,9 @@ const jwtPayload = {
   name: 'John Doe',
   iat: 1516239022,
   ext_attr: { enhancer: 'XSUAA' },
-  aud: ['clientid'],
-  zid: 'my-zone'
+  aud: ['xsapp-myapp!123.clientid'],
```

Comment:
client_id & audience have to match the mocked xsuaa credentials, that's what the change is doing